src/npg_porch/models/task.py

```diff
@@ -107,6 +107,11 @@ class TaskExpanded(Task):
         title="Task Created",
         description="The timestamp of task creation",
     )
+    updated: datetime = Field(
+        default=None,
```

Comment:
The way we query the db, we will always have this date. It might be the same date as the task creation date. However, the task creation date is also nullable. This does not seem to reflect reality. Probably review later.